### PR TITLE
 first code review

### PR DESCRIPTION
**Pull request by HanningtoneM**
introduction to the code review process to upgrade HTML and CSS skills.

### CODE REVIEW

**Comment by HanningtoneM** [comment]
- Good job so far
- there are still some issues to improve.
- your repository doesn't have a linters file. kindly include linters so you can check for the linters errors
- please add a CSS file to your code

**Comment by HanningtoneM** [comment]
- please add a descriptive readme file to document your repo more professionally
- kindly add more descriptive commit messages

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <div id="title">Dr. Norman Borlaug</div>
+      <figure id="img-div">
+        <img
+          src="https://merriam-webster.com/assets/mw/images/article/art-wap-landing-mp-lg/alt-5ae892611bf1a-5168-1472832016f2509f3889266323039a33@1x.jpg"
+          alt=""
```
Comment (zainsadaqat) [inline index.html:15]: #### Inline Comment Example
Please write an alternative text for this image 😄

**Review by zainsadaqat — APPROVED** [review]
# Hello @HanningtoneM  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

[Optional]

- [ ] Please make inline comments by directly pointing out the lines where you think there's a mistake. I'm leaving an inline comment for you in `index.html` file if you have any confusion.

[Optional]

- [ ] Please ask student to use imperative sentence for writing commit messages. 👍🏻 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**